src/Scanner.ts

```diff
@@ -60,6 +60,14 @@ class Scanner {
     }
   }
 
+  private addAttributeToken(
+    type: LexemeType.HTML_ATTRIBUTE,
```

Comment:
Attributes shouldn't be a new lexeme, but rather a property on `HTML_OPENING_TAG` and `HTML_SELFCLOSING_TAG`